srcpkgs/python3-chromecast/template

```diff
@@ -1,17 +1,17 @@
 # Template file for 'python3-chromecast'
 pkgname=python3-chromecast
-version=7.7.2
+version=8.0.0
 revision=1
 wrksrc="pychromecast-${version}"
 build_style=python3-module
-hostmakedepends="python3-setuptools"
+hostmakedepends="python3-setuptools python3-wheel python3-pip"
```

Comment:
No, this is not how you fix the package. You need to add things from https://github.com/home-assistant-libs/pychromecast/blob/master/requirements.txt (they are in depends) to hostmakedepends.

Also add https://github.com/home-assistant-libs/pychromecast/blob/master/requirements-test.txt to checkdepends, if we have packages for all of them.

Reply:
> No, this is not how you fix the package. You need to add things from https://github.com/home-assistant-libs/pychromecast/blob/master/requirements.txt (they are in depends) to hostmakedepends.
> 
> Also add https://github.com/home-assistant-libs/pychromecast/blob/master/requirements-test.txt to checkdepends, if we have packages for all of them.

Done. Thank you.